data_pump/utils.py

```diff
@@ -94,3 +95,23 @@ def save_dict_as_json(json_name, dictionary: dict):
 def create_dict_from_json(json_name):
     return {int(key): value
             for key, value in read_json(json_name, MAPPING_PATH).items()}
+
+
+def create_bitstream(bundle=None, name=None, path=None, mime=None, metadata=None):
```

Comment:
these specific functions should not be here

Reply:
Why? It should be in another place or it should be removed..?

Reply:
Yes, you are right. I realized that I made this feature from the non up to date `main` branch. I didn't have pulled `main` branch... but now it is up to date.